### DISCUSSION TITLE
Integration with the grafana

### DISCUSSION THREAD

**Discussion by kurdak** [SBFspot/SBFspot#495] [General]
Hello.
Thank you for your great job!
I was able to integrate your solution with Grafana, thanks to which it now has great charts. 
![obraz](https://user-images.githubusercontent.com/1711933/138760238-e27f1fbb-a0a2-4773-8285-6782be7e838e.png)
By executing the -startdate command, you only import a portion of the data without the voltages, or is there a way to do that as well? 
One more question, do you know anything about graphs in the device itself? Since using your solution, I have not seen daily charts there.
SMA Sunny Tripower.

**Comment by MBakula1** [comment]
Hello,
Can you explain how to integrate two or more topics from SBFspot to Grafana?
Best regards

**Comment by hevilp** [comment]
Just add the sqlite as datasource, then put following query:
`Select Dat.TimeStamp AS time, Power As value FROM DayData Dat ORDER BY Dat.Timestamp Desc`

If you did this, answer here, i can give you 2 other usefull graphs

**Comment by shadowbyte68** [comment]
`Select Dat.TimeStamp AS time, Power As value FROM DayData Dat ORDER BY Dat.Timestamp Desc`

don't work ... :(

**Comment by EuroAfrican** [comment]
This worked great for me, what else did you make Grafana plot?
